bilby/gw/detector/calibration.py

```diff
@@ -27,6 +91,13 @@ def read_calibration_file(filename, frequency_array, number_of_response_curves,
     starting_index: int
         Index of the first curve to use within the array. This allows for segmenting the calibration curve array
         into smaller pieces.
+    correction_type: str
+        How the correction is defined, either to the :code:`data`
+        (default) or the :code:`template`. In general, data products
+        produced by the LVK calibration groups assume :code:`data`.
+        The default value will be removed in a future release and
+        this will need to be explicitly specified.
+        ..versionadded XYZ
```

Comment:
You were probably already planning on changing this, but reminder to change this to the actual version.